src/ImageSharp.Web/Middleware/ImageWorkerResult.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.Web.Resolvers;
+
+namespace SixLabors.ImageSharp.Web.Middleware
+{
+    /// <summary>
+    /// Provides an asynchronous worker result.
+    /// </summary>
+    internal struct ImageWorkerResult
```

Comment:
This should be a readonly struct with any methods accepting the type as a parameter passed via `in` to prevent a copy.

Reply:
Done. It's only a holder, so not passed anywhere that can take an `in`